### PR TITLE
fixed duplicate use of cbor tag 560

### PR DESCRIPTION
**Pull request by nedmsmith**
changed $crypto-keys-type-choice to use 561. Fixes issue #132

### CODE REVIEW

**Comment by babongo** [comment]
Thanks @setrofim for spotting and @nedmsmith for the patch.

I'd go further here and add a bit of logics to the pipeline to automatically check for duplications, so that we don't end up in the same situation in the future.

**Comment by nedmsmith** [comment]
I'm open to suggestions for how to do that.

**Comment by thomas-fossati** [comment]
> I'm open to suggestions for how to do that.

Whenever we add a new tag we should re-run the `cbor-tags.txt` target to update the tags catalogue.

We could couple that with a check for duplicates in the `cbor-tags.txt` file, e.g.:
```
grep -E -h -o '#6\.[0-9]{3}' cbor-tags.txt | uniq -cd
```
and exit non-zero if there's at least one line.

**Review by setrofim — APPROVED** [review]

File: cddl/Makefile [hunk 1]
```diff
@@ -57,3 +58,9 @@ clean: ; rm -f $(CLEANFILES)
 
 # Extract the CBOR tags defined by CoRIM/CoMID (i.e., those in the 5xx space)
 cbor-tags.txt: $(wildcard *.cddl) ; grep -h '#6\.5' *cddl | sort -u -t'=' -k2 > $@
+
+# Return an error if there are duplicat CBOR Tags defined
+cbor-tags-unique: $(wildcard *.cddl) ; \
+	 grep -h '#6\.5' *cddl | sort -u -t'=' -k2 > cbor-tags.txt ; \
```
Comment (thomas-fossati) [inline cddl/Makefile:64]: I guess this target should depend explicitly on `cbor-tags.txt` -- this way L64 becomes redundant

File: cddl/Makefile [hunk 2]
```diff
@@ -57,3 +58,9 @@ clean: ; rm -f $(CLEANFILES)
...
+	 grep -E -h -o '#6\.[0-9]{3}' cbor-tags.txt | uniq -cd ; \
+	 if [ ! -z $1 ]; then exit 1; else echo "CBOR tags are unique"; fi
```
Comment (thomas-fossati) [inline cddl/Makefile:66]: `$1` is a positional parameter, I don't think this will expand to anything useful.

**Review by thomas-fossati — COMMENTED** [review]
Thanks for taking a stab!  I'll provide a couple of tweaks

**Review by thomas-fossati — APPROVED** [review]

**Comment by nedmsmith** [comment]
LGTM

**Pull request merged**